docs/index.mdx

```diff
@@ -46,6 +46,22 @@ Second, a GitPOAP is awarded - Contributors are awarded GitPOAPs (POAPs with an
 
 <HowSvg />
 
+### Building on GitPOAP
+
+On-chain identities are only useful if there are others to interact with and applications built to leverage them.  That’s why we decided to build on top of POAP.  
```

Comment:
On-chain identities are only useful if there are others people to interact with and applications built to leverage them. That’s why we've decided to build on top of POAP.